### ISSUE TITLE
Want to be able to adjust the effective spacing between cell rows without effecting spacing between columns

### ISSUE THREAD

**Issue by jem5519** [techprimate/TPPDF#151]
When using a table to lay out a report I would like to reduce the spacing between cell rows without effecting the spacing between cell columns. I understand that this may be achieved by changing the top and bottom padding of cells independent of the padding on the sides, but I don't see any way to do this in the current exposed commands.

**Comment by techprimate-phil** [comment]
Yes, currently there is only one `padding` and one `margin` value to set these values for all directions.

We could add individual values per side, but these would be horizontal values per column, and vertical values per row and not for individual cells.

**Comment by jem5519** [comment]
Hi Phillip,

Thank you for responding. That behavior is exactly what I would want: the setting would affect the whole row.


- Jim

**Comment by Vinzius** [comment]
Hello !

Just in case it could help someone looking for a solution, I bypassed the current limitation by adding a minimum line height to an attributed string.

Of course, it'll only work with text components, but it's a solution in case you really need the padding ^^

So for instance:
```swift
let font = UIFont.boldSystemFont(ofSize: 12)
let pStyle = NSMutableParagraphStyle()
pStyle.minimumLineHeight = ceil(font.lineHeight * 1.7)

NSAttributedString(string: "My text", attributes: [
  .font: font,
  .foregroundColor: UIColor.black,
  .paragraphStyle: pStyle
])
```

(by the way, thanks a lot for the lib!)
